### ISSUE TITLE
Gantt chart bug: weird interaction with notes, tasks and separators

### ISSUE THREAD

**Issue by RohanTalip** [plantuml/plantuml#386]
I'm seeing a weird interaction with notes, tasks and separators:

```
@startgantt

Project starts 2020-09-01

[taskA] starts 2020-09-01 and lasts 3 days
[taskB] starts 2020-09-10 and lasts 3 days
[taskB] displays on same row as [taskA]

[task01] starts 2020-09-05 and lasts 4 days

then [task02] lasts 8 days
note bottom
  note for task02
  more notes
end note

then [task03] lasts 7 days
note bottom
  note for task03
  more notes
end note

-- separator --

[taskC] starts 2020-09-02 and lasts 5 days
[taskD] starts 2020-09-09 and lasts 5 days
[taskD] displays on same row as [taskC]

[task 10] starts 2020-09-05 and lasts 5 days
then [task 11] lasts 5 days

@endgantt
```

[uml](http://www.plantuml.com/plantuml/uml/XPBDJiCm48JlVefv0KjsGWMz5Se3S4SvBCJyGEEjl2iXtXuhJYLNLi8jcPbygndb84f9tocg6lEK-2kyAcPDqBdEMRUtpXlph2JVzqDhWUA84qaMUupqAoNwq4IzMu-Edt8-PG4S8JG59Fu12PQhotdds_kx2dfJe4O_GaJveXiMxwvuaJNWXLLvCi3yziO99PkLYLEOPJ4XZlFJ5Q-_y6x_v_LhF6iXuKo9D4UjNTeTst9TLMvNBVROHlVhqSrnZvTnuTtskeLQZG7lXslF77B9yY_z0G00)
![overlapping](https://user-images.githubusercontent.com/7445140/94315751-01b34a00-ff38-11ea-9a40-700943ac74ab.png)

Things I'm noticing about this:

1. the separator is not pushed down by the note for task03 (or not enough)
1. taskC and taskD are pushed down by the note for task03
1. task10 and task11 are not pushed down by the note for task03
1. taskC and taskD are now overlapped by task10

Here is what it looks like with the notes commented out ([uml](http://www.plantuml.com/plantuml/uml/ZPBDJiCm48JlVegvzMJ9JgYWjq9v0EueXuMO7sZiohiIukqnjYkbiXBrbinycTMCicUXAE_aHPHwYk7BlGgonkXCPxJPQMELUXRYx_kXjK1-n94uYJr6-kM2FZIeDSle-CcdOn8GF9WcXnX-G8nwjPutjhs_dONUb50b7ywZVD4DrRihtiO7SNW98c5I6o2_leM8GlzBKuWkwubsVioF5vdzEVFscin-ELDhi3jH94cmrhNbeItPpKfkPyizjkXk4LqV-N0U6TQihbnJPuF0sk7IK_jKClzJVm00)) :
![not-overlapping](https://user-images.githubusercontent.com/7445140/94316125-a46bc880-ff38-11ea-89f6-609049477641.png)

I see something similar noted about overlapping on https://plantuml.com/gantt-diagram#c5aeee17b294caf8 :
![Screen Shot 2020-09-25 at 2 12 36 PM](https://user-images.githubusercontent.com/7445140/94316451-37a4fe00-ff39-11ea-8160-8983f5194dd3.png)

I thought that having another test case might be useful.

**Comment by RohanTalip** [comment]
It looks like this got fixed in [V1.2020.18](https://plantuml.com/changes) (released 2020-09-30).

Thanks!

**Comment by schangze** [comment]
So what's the difference? when I used version 2026, it still overlaps
